velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -53,6 +54,28 @@ class PrefixSortEncoder {
     }
   }
 
+  /// Encode String types.
+  FOLLY_ALWAYS_INLINE void encode(
+      std::optional<StringView> value,
+      char* dest,
+      int32_t encodeSize) const {
+    if (value.has_value()) {
+      dest[0] = nullsFirst_ ? 1 : 0;
```

Comment:
Could you provide more information to prove that StringView Compare and memcompare are equivalent?

Reply:
Row container compares null bits first, https://github.com/facebookincubator/velox/blob/ad27145c26980f65768d65fc65261082eeb2a933/velox/exec/RowContainer.h#L1190
Then it calls StringVIew.compare to get the comparsion result. https://github.com/facebookincubator/velox/blob/ad27145c26980f65768d65fc65261082eeb2a933/velox/type/StringView.h#L159.
The prefix is null byte + string content + padding zero. We follow similar way to padd zero like `prefix_` in StringView, if it's shorter than max value. 
And I updated test cases to make sure they are equivalent.